web/src/search_suggestion.ts

```diff
@@ -402,7 +440,7 @@ function get_topic_suggestions(last, terms) {
             if (filter.has_operator("stream")) {
                 stream = filter.operands("stream")[0];
             } else {
-                stream = narrow_state.stream_name();
+                stream = narrow_state.stream_name() ?? "";
```

Comment:
Before this could be `undefined` and here you're setting it to be the empty string. Was it a bug before? Or should it stay undefined here. This kind of functional change would be helpful as a separate commit with a message explaining the change.

Reply:
Thanks for pointing out this. I missed this, because this doesn't raise any bug in the code. In the case of stream being `undefined`  an empty array should be returned, which we are doing ultimately in line 449. That conditional should be here also and the empty array should be returned from here( or maybe just adding a normal condition to not to push is pretty much possible) instead of pushing an empty string unnecessarily to `suggest_terms` which wont be used anyways.
I will fix that.
Thanks.